CONTRIBUTING.md

```diff
@@ -32,20 +32,20 @@ We recommend creating a folder somewhere manually called `node_modules`. For exa
+How do you get dat to use the git copy of hyperdrive? Just delete the npm registry copy!
 
 ```
 rm -rf ~/code/node_modules/dat/node_modules/hyperdrive
 ```
 
 Now when you run dat, and it tries to `require('hyperdrive')` it first looks in its own `node_modules` folder at `~/code/node_modules/dat/node_modules` but doesnt find hyperdrive. So it goes up to `~/code/node_modules` and finds `hyperdrive` there and uses that one, your git copy.
 
-If you want to switch back to an npm copy, just run `npm install` inside `~/code/node_modules/dat/` and npm will download any missing modules into `~/code/node_modules/dat/node_modules` but wont touch anything in `~/code/node_modules`.
+If you want to switch back to npm registry copy, just run `yarn install` or `npm install`  inside `~/code/node_modules/dat/` and any missing modules will be downloaded into `~/code/node_modules/dat/node_modules` without touching anything in `~/code/node_modules`.
 
 This might seem a bit complicated at first, but is simple once you get the hang of it. Here are some rules to help you get started:
 
-- Never make any meaningful edits to code inside an "npm-managed" node_modules folder (such as `~/code/node_modules/dat/node_modules`), because when you run `npm install` inside those folders it could inadvertently delete all of your edits when installing an updated copy of a module. This has happened to me many times, so I just always use my git copy and delete the npm copy (as described above) to make edits to a module.
-- You should never need to run any npm commands in terminal when at your "manually managed"" node_modules folder at `~/code/node_modules`. Never running npm commands at that folder also prevents npm from accidentally erasing your git copies of modules
+- Never make any meaningful edits to code inside an "npm/yarn-managed" node_modules folder (such as `~/code/node_modules/dat/node_modules`), because any changes inside those folders could inadvertently be deleted when installing an updated copy of a module. This has happened to me many times, so I just always use my git copy and delete the npm registry copy (as described above) to make edits to a module.
+- You should never need to run any npm/yarn commands in terminal when at your "manually managed"" node_modules folder at `~/code/node_modules`. Never running npm/yarn commands at that folder also your git copies of the modules from getting erased.
 - The location of your "manually managed" node_modules folder should be somewhere isolated from your normal require path. E.g. if you put it at `~/node_modules`, then when you run `npm install dat` at `~/Desktop` npm might decide to erase your git copy of dat at `~/node_modules/dat` and replace it with a copy from npm, which could make you lose work. Putting your manually managed `node_modules` folder in a sub-folder like `~/code` gets it "out of the way" and prevents accidents like that from happening.
```

Comment:
`npm install` language still used here

Reply:
I was hesitant to remove it from this paragraph because it's discussing working on dependencies, and really when working on a dependency you should be using whichever package manager that project uses (whether it's yarn, npm, pnpm, etc.) The project being discussed in this case is `dat`, which does use `npm`.

It certainly could be explained better - the switching back and forth is a little confusing. I was hesitant to spend too much effort on this though, as this workflow likely isn't what we want to encourage anymore. `npm link` offers basically the same advantages, plus it's a more commonly used workflow and it offers slightly better guarantees in terms of dependency deduplication. The workflow described here still works just fine, but it's the older way of doing things, and I think it's a bit harder to understand. The only distinct advantage I can think of is that it avoids symlinks, which can be problematic on certain environments.

Reply:
I would vote for removing this section outright, it doesn't quite need to be here

Reply:
Fair enough - I've removed it.